scripts/tasks/chromatic.ts

```diff
@@ -8,19 +8,17 @@ export const chromatic: Task = {
   async ready() {
     return false;
   },
-  async run({ key, sandboxDir, builtSandboxDir, junitFilename }, { dryRun, debug }) {
+  async run({ key, scriptsDir, builtSandboxDir }, { dryRun, debug }) {
     const tokenEnvVarName = `CHROMATIC_TOKEN_${key.toUpperCase().replace(/\/|-|\./g, '_')}`;
     const token = process.env[tokenEnvVarName];
 
-    await exec(
-      `npx chromatic \
-          --debug \
-          --exit-zero-on-changes \
-          --storybook-build-dir=${builtSandboxDir} \
-          --junit-report=${junitFilename} \
-          --projectToken=${token}`,
-      { cwd: sandboxDir },
-      { dryRun, debug }
-    );
+    const command = `yarn chromatic`;
+    const flags = [
+      `--storybook-build-dir=${builtSandboxDir}`,
+      `--exit-once-uploaded`,
+      `--projectToken=${token}`,
+    ];
+
+    await exec(`${command} ${flags.join(' ')} `, { cwd: scriptsDir }, { dryRun, debug });
```

Comment:
We could actually use the chromatic node API for this now 🤷

Reply:
Yes, if it's stable, we should, I think.